ceph.spec.in

```diff
@@ -131,6 +131,7 @@ BuildRequires:	python-pecan
 BuildRequires:	python-requests
 BuildRequires:	python-virtualenv
 BuildRequires:	python-werkzeug
+BuildRequires:	socat
```

Comment:
maybe we should just move the build-depends for readying "make check" into `install-deps.sh`.